spyder/widgets/mixins.py

```diff
@@ -53,7 +53,7 @@ class BaseEditMixin(object):
     _DEFAULT_TITLE_COLOR = '#2D62FF'
     _CHAR_HIGHLIGHT_COLOR = 'red'
     _DEFAULT_TEXT_COLOR = '#999999'
-    _DEFAULT_LANGUAGE = 'python'
+    _DEFAULT_LANGUAGE = ''

```

Comment:
I think we should use `'python'` here. Why did you decide to leave it empty?

Reply:
I was checking the use of `getattr` for trying to get the `language` and the default values used are the empty `string` or `None`, should I revert the change and use `python` instead?

Reply:
Yeah, please do. I think it'd be easier to understand the code that way.